docs/source/hints.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Composition hints
+---
+
+When you successfully [compose](./federated-types/composition) the schemas provided by your [subgraphs](./building-supergraphs/subgraphs-overview/) into a supergraph schema, the composition process can flag potential improvements. Specifically, composition can output violations of the [GraphOS schema linter's](/graphos/delivery/schema-linter) [composition rules](/graphos/delivery/linter-rules#composition-rules) in the GraphOS Studio or via the [Rover CLI](/rover/). Both [`rover supergraph compose`](/rover/commands/supergraphs#supergraph-compose) and [`rover subgraph check`](/rover/commands/subgraphs#subgraph-check) commands output rule violations with the [severity levels](/graphos/delivery/schema-linter/#setting-severity-levels) you've configured for those rules.
```

Comment:
```suggestion
When you successfully [compose](./federated-types/composition) the schemas provided by your [subgraphs](./building-supergraphs/subgraphs-overview/) into a supergraph schema, the composition process can flag potential improvements. Specifically, composition can output violations of the [GraphOS schema linter's](/graphos/delivery/schema-linter) [composition rules](/graphos/delivery/linter-rules#composition-rules) in the GraphOS Studio or via the [Rover CLI](/rover/). Both [`rover supergraph compose`](/rover/commands/supergraphs#supergraph-compose) and [`rover subgraph check`](/rover/commands/subgraphs#subgraph-check) commands output rule violations with the [severity levels](/graphos/delivery/schema-linter/#setting-severity-levels) you've configured for those rules.
```

```suggestion
When you successfully [compose](./federated-types/composition) the schemas provided by your [subgraphs](./building-supergraphs/subgraphs-overview/) into a supergraph schema, the composition process can flag potential improvements. Specifically, composition can output violations of the [GraphOS schema linter's](/graphos/delivery/schema-linter) [composition rules](/graphos/delivery/linter-rules#composition-rules) in GraphOS Studio or via the [Rover CLI](/rover/). The [`rover subgraph check`](/rover/commands/subgraphs#subgraph-check) command outputs rule violations with the [severity levels](/graphos/delivery/schema-linter/#setting-severity-levels) you've configured for your graph variant. [`rover supergraph compose`](/rover/commands/supergraphs#supergraph-compose) outputs all rule violations. 
```

Don't love my wording here but it's important to distinguish between `rover subgraph check` which is associated with a given graph variant in Apollo Studio, and `rover supergraph compose` which is _not_ associated with a given graph variant in Apollo Studio, instead running composition on subgraph schemas local to the machine the command is being run on (it does not make any network requests or require an Apollo API key like the `rover subgraph check` command does.)

Reply:
Good distinction, thanks!